src/vs/workbench/parts/search/common/searchModel.ts

```diff
@@ -40,7 +40,7 @@ export class Match {
 	private _fullPreviewRange: ISearchRange;
 
 	constructor(private _parent: FileMatch, private _fullPreviewLines: string[], _fullPreviewRange: ISearchRange, _documentRange: ISearchRange) {
-		this._oneLinePreviewText = _fullPreviewLines[_fullPreviewRange.startLineNumber];
+		this._oneLinePreviewText = stripUTF8BOM(_fullPreviewLines[_fullPreviewRange.startLineNumber]);
```

Comment:
Why this?

Reply:
But the BOM should be stripped already by ripgrepTextSearchEngine right?

Reply:
I can move it to ripgrepTextSearchEngine.

Reply:
But it's already there, right? That file should be stripping the BOM correctly in all cases with your change.